.github/PULL_REQUEST_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,53 @@
+## Description
+<!-- Briefly describe the bug you are trying to fix -->
+
+<!-- Describe your changes in detail -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## Checklist
+<!-- Feel free to remove check-list items aren't relevant to your change -->
+
+- [ ] Is this PR following the [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
+      PR format (described below)?
+- [ ] Have you added the unit tests to prevent issue reoccurance( using [pytest fixture pattern](
+      (https://docs.pytest.org/en/latest/fixture.html#fixture))?
+- [ ] Have you run the unit tests using pytest?
+- [ ] Is your code style correct (are you using PEP8, pylint)?
+
+<!--
+Thank you so much for your PR!  To help us review your contribution, please
+consider the following points:
+
+- Read contributing guidelines carefully before submitting the PR. It is available
+  at https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md.
+
+- Read and abide by the Code of Conduct. It is available at  
+  https://github.com/arviz-devs/arviz/blob/master/CODE_OF_CONDUCT.md
+
+- Do not create the PR out of master, but out of a separate branch.
+
+- The PR title should summarize the changes, for example "Add new group argument for the
+  pair plot".  Avoid non-descriptive titles such as "Addresses issue #348". If your pull 
+  request addresses an issue, please use the pull request title to describe 
+  the issue and mention the issue number in the pull request description.
+
+- The description should provide at least 1-2 sentences describing the pull request
+  in detail (Why is this change required?  What problem does it solve?) and
+  link to any relevant issues. If modifying a plot, render your plot to inspect for changes 
+  and copy image in the pull request message on Github
+
+- Please prefix the title of incomplete contributions with [WIP] (to indicate a work in 
+  progress). WIPs may be useful to (1) indicate you are working on something to avoid 
+  duplicated work, (2) request broad review of functionality or API, or (3) seek collaborators.
```

Comment:
I would merge this with the comments above in the PR descriptions section.

Reply:
Yes, that would make sense.

Reply:
This looks very good, now I would probably remove the lines below about breifly describe the bug, describing the changes in detail, linking to issue as it is already here.